Session 5/arrayHelpers.js

```diff
@@ -0,0 +1,48 @@
+//http://www.codewars.com/kata/array-helpers
+
+Array.prototype.square = function () {
+    var result = [];
+    for (let i = 0; i < this.length; i++) {
```

Comment:
try to use built in array methods to solve this code wars problem instead of `for` loops.  also, you can use the new methods you define in this file for defining other methods